python/artm/artm_model.py

```diff
@@ -582,10 +574,10 @@ def _func(batch):
 
         self._wait_for_batches_processed(
             self._pool.apply_async(func=self.master.fit_online,
-                                   args=(batches_list, batch_vectorizer.weights,
+                                   args=(batch_vectorizer.batches_ids, batch_vectorizer.weights,
                                          update_after_final, apply_weight_final,
                                          decay_weight_final, async)),
-            len(batches_list))
+            len(batch_vectorizer.batches_ids))
```

Comment:
Is it OK to write `__len__` function to count number of entities in `batch_vectorizer` instead `num_batches` property and even ugly `len(batch_vectorizer.batches_ids)`, like in file `Scores` class from `scores.py`?